### ISSUE TITLE
[SCIM] Sorting doesn't work in MongoDB 

### ISSUE THREAD

**Issue by gabrielemilan** [simpleidserver/SimpleIdServer#408]
Hi guys,

I notice that the sortBy and sortOrder attribute don't work in Mongo db version.

Regards.

**Comment by simpleidserver** [comment]
Hello,

First thank you for your feedback :)
The issue is fixed in the branch `release/v2.0.26`, the sorting feature were missing in the Mongo DB version.

KR,

SID

**Comment by simpleidserver** [comment]
A pre-release package is available with the fix 

```
dotnet add package SimpleIdServer.Scim.Persistence.MongoDB --version 2.0.26-ci-00398 --source https://www.myget.org/F/advance-ict/api/v3/index.json
dotnet add package SimpleIdServer.Scim --version 2.0.26-ci-00398 --source https://www.myget.org/F/advance-ict/api/v3/index.json
```
